CHANGELOG.md

```diff
@@ -3,13 +3,17 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and
 [Keep a changlelog's changelog standard](http://keepachangelog.com/)
 
+## [Unreleased](https://github.com/FredrikNoren/ungit/compare/v0.10.3...master)
+
 ### Added
 - New configuration option `logLevel` allows you to assign the level of logging you want to see in the servers output console.
 - New configuration option `mergeTool` allows you to assign a custom external merge tool for conflict resolution [783](https://github.com/FredrikNoren/ungit/issues/783)
-
-## [Unreleased](https://github.com/FredrikNoren/ungit/compare/v0.10.3...master)
+- Whitespace ignore option for text diffs [777](https://github.com/FredrikNoren/ungit/issues/777)
```

Comment:
Nit: use hashtag before issue number for consistency. (The one above this also doesn't have it...)